### PR TITLE
Pronoy/mobile slo widget

### PR DESCRIPTION
**Pull request by PronoyC**
### What does this PR do?
This PR introduces the SLOs mobile widget by providing screenshots and a brief description, along with some minor consistency changes on the mobile app page.

### Motivation
The SLOs mobile widget will be GA soon so this is in preparation for this.

### Preview

 
https://docs-staging.datadoghq.com/pronoy/mobile-slo-widget/mobile/?tab=ios#widgets

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/mobile/_index.md [hunk 1]
```diff
@@ -80,11 +80,30 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+Timeframe options are:
+- 7 days
+- 30 days
+- 90 days
+- Previous week
+- Previous month
+- Week to date
+- Month to date
+
+You can also choose which dashboard from your organization opens in the app when you tap on an SLO widget, allowing you to quickly dig deeper into your metrics.
```
Comment (kayayarai) [inline content/en/mobile/_index.md:98]:
```suggestion
You can also choose a dashboard to open when you tap on an SLO widget, allowing you to quickly dig deeper into your metrics.
```

File: content/en/mobile/_index.md [hunk 2]
```diff
@@ -80,11 +80,30 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+
+**Note**: If you do not choose a dashboard to open in the app when you tap on an SLO widget, the widget will simply open the app by default.
```
Comment (kayayarai) [inline content/en/mobile/_index.md:100]:
```suggestion
**Note**: If you do not specify a dashboard to open, tapping the SLO widget opens the Datadog app.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Just a couple wording changes for clarity and style. Let me know if I changed your intended meaning at all.

File: content/en/mobile/_index.md (hunk 1 above)
Comment (PronoyC) [reply content/en/mobile/_index.md:98]: updating copy to suggested change but adding a few clarifying words

**Comment by PronoyC** [comment]
So the image we are using for the release is slightly outdated so opting to close the PR for now until we get that confirmed. I have committed the suggested changes (with some minor tweaks)

**Pull request closed without merging**